ert3/config/_parameters_config.py

```diff
@@ -147,7 +147,10 @@ def _ensure_variables(cls, variables):  # type: ignore
         if len(variables) > 0:
             return variables
 
-        raise ValueError("Parameter group cannot have no variables")
+        raise ValueError(
+            "Parameter group cannot have empty variable list.\n"
+            "Avoid specifying variables to get scalars. "
```

Comment:
```suggestion
            "Avoid specifying variables to get scalars."
```

This would probably keep you up at night…

Reply:
It would! :)